platform/tests/EducationBenchmarking.Platform.Tests/Establishment/WhenFunctionReceivesSuggestOrganisationsRequest.cs

```diff
@@ -0,0 +1,61 @@
+using EducationBenchmarking.Platform.Api.Establishment.Models;
+using EducationBenchmarking.Platform.Infrastructure.Search;
+using EducationBenchmarking.Platform.Shared;
+using FluentValidation.Results;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using Xunit;
+
+namespace EducationBenchmarking.Platform.Tests.Establishment;
+
+public class WhenFunctionReceivesSuggestOrganisationsRequest : OrganisationsFunctionsTestBase
+{
+    [Fact]
+    public async Task ShouldReturn200OnValidRequest()
+    {
+        Search
+            .Setup(d => d.SuggestAsync(It.IsAny<PostSuggestRequest>(), It.IsAny<CancellationToken>()))
+            .ReturnsAsync(new SuggestOutput<Organisation>());
+
+        Validator
+            .Setup(v => v.ValidateAsync(It.IsAny<PostSuggestRequest>(), It.IsAny<CancellationToken>()))
+            .ReturnsAsync(new ValidationResult());
+
+        var result =
+            await Functions.SuggestOrganisationsAsync(CreateRequestWithBody(new PostSuggestRequest())) as JsonContentResult;
+
+        Assert.NotNull(result);
+        Assert.Equal(200, result?.StatusCode);
+    }
+
+    [Fact]
+    public async Task ShouldReturn40OOnInvalidRequest()
```

Comment:
there is a typo in the method name in 400.

Reply:
Pushed a fix